packages/frontend/src/components/molecules/Header.tsx

```diff
@@ -0,0 +1,53 @@
+'use client';
+import React, { useState, useEffect } from 'react';
+import { FaBars } from 'react-icons/fa';
```

Comment:
are we using this FaBar or the one in the nav bar?

Reply:
navbar is the one that should be responsible of handling the responsiveness task of whether showing the fa bar or not based on whether we are in mobile screen or not